### ISSUE TITLE
Release of V1.5

### ISSUE THREAD

**Issue by makotok** [moai/moai-dev#785]
It has not released a while Moai SDK.
Schedule and release goals V1.5 What do you do?

**Comment by halfnelson** [comment]
@makotok what else is outstanding before we can launch 1.5? Do you have a checklist somewhere of what needs to happen?

**Comment by halfnelson** [comment]
It might be an idea to tie the release to the new website launch? so we can get people previously scared off back again with a big announcement.

**Comment by makotok** [comment]
Currently, there is no checklist. 
Soon, I want to create a tag for V1.5.1 officially. 

What should be done about the following points? 
- Binary release (Windows, MacOSX, Linux) 
- API documentation 
- Release Script 

Whatever the case may be, would be to create a tag once.

**Comment by ibisum** [comment]
> @makotok what else is outstanding before we can launch 1.5? Do you have a checklist somewhere of what needs to happen?

I will have some last-minute contributions to check in about the middle of the week, involving the Camera, both for iOS and Android .. i.e. “MOAIApp.getCameraPicture()” that will return the texture to MOAI for use ..

j.

**Comment by frantufro** [comment]
Guys, I've done a lot of work on release scripting and API parsing, there were some pending things but you can find all of that in https://github.com/moai/moai-dev/tree/master/bin
See distribute.sh

**Comment by makotok** [comment]
@franciscotufro 
Thank you for information. 
However, the structure of the libraries and header files has changed greatly. 
As a result, the host of the binary was no longer working. 

The required binary release?
I think it is okay to maintenance slowly.

**Comment by makotok** [comment]
I have created a tag for the time being. 
It is a tag after a long time. 

Build script creates based on distribute.sh. 
CI server does not exist. 
Therefore, it will be able to build on each OS.

**Comment by DanielSWolf** [comment]
@makotok Nice to have an official release again. However, I suggest that future releases be announced more than one day in advance. There's a number of small fixes I would have liked to add before the release.

**Comment by halfnelson** [comment]
In order to help finish this off. I have a gh-pages branch on moaiforge with the latest documentation built. http://moaiforge.github.io/moai-sdk/api/latest. I can't generate a pull request since moai-dev does not have the require gh-pages detached branch. So I am leaving this note.

**Comment by halfnelson** [comment]
OK, what is left holding this up, is it pre-compiled host builds?

**Comment by makotok** [comment]
@halfnelson 
Thank you for information. 
I try.

**Comment by makotok** [comment]
To be closed because it was released.
